spec/support/pages/messages/create.rb

```diff
@@ -44,7 +44,7 @@ def add_text(text)
       find('.ck-content').base.send_keys text
     end
 
-    def click_save
+    def click_save(expect_success: true)
```

Comment:
@HDinger @oliverguenther I'm not sure I understand this change https://github.com/opf/openproject/pull/13660/commits/84cb848e418a7e599aa531388b17822581375a42 as it doesn't seem to change any behaviour 🤔

Reply:
ping on this one @HDinger 😄

Reply:
you can remove it, it was part of a temporary fix and I seem to have overlooked that part